Dockerfile

```diff
@@ -32,10 +27,18 @@ RUN pip install --no-cache-dir torch torchvision torchaudio --index-url https://
     pip install --no-cache-dir pytorch-lightning==1.9 --no-deps && \
     pip install --no-cache-dir gdown
 
-RUN conda init bash && \
-    echo "conda activate splat-slam" >> ~/.bashrc
+FROM pytorch/pytorch:2.1.0-cuda11.8-cudnn8-runtime
+ENV DEBIAN_FRONTEND=noninteractive
 
-COPY download_pretrained_model.sh /usr/local/bin/download_pretrained_model.sh
+RUN apt update && \
+    apt install -y --no-install-recommends tzdata git unzip wget libglib2.0-0 libgl1-mesa-glx && \
```

Comment:
`tzdata git unzip wget` could be removed

Reply:
also adding ` --no-install-suggests` could help to reduce space, so:
`apt install -y --no-install-recommends --no-install-suggests libglib2.0-0 libgl1-mesa-glx`

Reply:
Good point! Maybe unzip is still needed because of unzipping the weights, but I'm not certain if the base image already includes it.